SynRD/benchmark/benchmark.py

```diff
@@ -54,5 +54,33 @@ def real_vs_private_soft_findings(self, paper):
         return 'Soft findings', percentage 
 
     
+def plot_example():
```

Comment:
It wasn't clear where to put it in Benchmark class, hence I left it here such that it's easy to find when needed.